Xuan p patch 1

xuanP opened the pull request:
Bonjour
Je décris un peu sur le travail dans README.txt. 
L 'article suggère d'implémenter 2 modèles (implicit et explicit avec AOS) et le code pour explicit n'est pas encore ’uploaded‘ maintenant.
Je maintenant fais la modification avec drlse.m qui est utilisé par easy_seg.m, donc j'ai pas uploaded drlse.m

xuanP committed:
FastSemiImplicitAOS Model

Bonjour
Je décris un peu sur le travail dans README.txt. 
L 'article suggère d'implémenter 2 modèles (implicit et explicit avec AOS) et le code pour explicit n'est pas encore ’uploaded‘ maintenant.

xuanP committed:
Delete drlse.m

xuanP committed:
Update README.txt

xuanP committed:
Delete essai.txt

xuanP committed:
Delete test.txt

xuanP committed:
Update easy_seg.m

troussil commented:
Plusieurs remarques: 
- faire un répertoire séparé
- l'objectif est d'implémenter la méthode de l'article, et non d'utiliser le code tout fait d'une autre méthode

xuanP committed:
Create AOS.m

xuanP committed:
Delete logTransformation.m

xuanP committed:
Delete easy_seg.m

xuanP committed:
Delete readImage.m

xuanP committed:
Delete images.jpg

xuanP committed:
Delete README.txt

xuanP committed:
Merci de votre remarque et une petitte question

nouveau .m: (des partie d'article formule 6~14, mais je travaille encore sur queques fonctions)
*stopFunction.m
*AOS.m
Oui je me suis rendu compte qu'il devrait être comme le geometric/geodesic active contour model traditionnel après que je commence à implémenter la 3ème Partie de Mise en Application Numérique. Parce que dans la 2ème partie je pensais que cette méthode AOS peut être adaptée à différents modèles tant que c'est implicite comme level-set based 
Je reviendrai à l'article. Mais est-ce possible pour moi de combiner l'AOS avec d'autres méthodes qui sont plus avancées?

xuanP commented:
nouveau .m: (des partie d'article formule 6~14, mais je travaille encore sur queques fonctions)
*stopFunction.m
*AOS.m
Oui je me suis rendu compte qu'il devrait être comme le geometric/geodesic active contour model traditionnel après que je commence à implémenter la 3ème Partie de numerical implementation. Parce que dans la 2ème partie je pensais que cette méthode AOS peut être adaptée à différents modèles tant que c'est implicite comme level-set based. Donc j'ai trouvé une méthode très jolie pour implementer l'AOS.
Je reviendrai à l'article. Mais est-ce possible pour moi de combiner l'AOS avec d'autres méthodes qui sont plus avancées?
